src/store/modules/users.js

```diff
@@ -148,11 +148,11 @@ const actions = {
     }
     commit('SET_CURRENT_ORG', systemOrg)
   },
-  leaveSettingOrg({ commit }) {
-    const preOrg = store.state.users.preOrg
-    if (!preOrg) {
-      return
-    }
+  leaveSettingOrg({ commit, state }) {
+    const preOrg = getPreOrgLocal(state.profile.username)
+
+    if (!preOrg) return
+
     commit('SET_CURRENT_ORG', preOrg)
   },
   enterGlobalOrg({ commit }) {
```

Comment:
There seem to be no major syntax or style inconsistencies in this snippet of code. However, it's best practice to use object properties instead of strings. Here is an updated version:

```javascript
const actions = {
  SET_CURRENT_ORG: 'SET_CURRENT_ORG',
};

function commit(key, payload) {
    // logic 
}
```

Remember, when using `commit` action, make sure its type matches with the expected input format.

The changes made ensure that all values passed into these functions will actually have the expected keys and values, improving future maintenance efforts.